### PR TITLE
Add pootle_fs.ResponseDisplay classes

### PR DESCRIPTION
**Pull request by phlax**
to handle parsing state/responses from actions

### CODE REVIEW

File: pootle/apps/pootle_fs/display.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+FS_EXISTS = [
+    "pulled_to_pootle", "pushed_to_fs", "fetched_from_fs",
+    "staged_for_merge_fs", "staged_for_merge_pootle",
+    "merged_from_fs", "merged_from_pootle"]
+
+STORE_EXISTS = [
+    "pulled_to_pootle", "pushed_to_fs", "added_from_pootle",
+    "staged_for_merge_fs", "staged_for_merge_pootle",
+    "merged_from_fs", "merged_from_pootle"]
```
Comment (unho) [inline pootle/apps/pootle_fs/display.py]: Just for simplifying maintaning this I would add a `BOTH_EXIST`, even if it is only used to then define `FS_EXISTS` and `STORE_EXISTS`:

```python
BOTH_EXIST = [
    "pulled_to_pootle",
    "pushed_to_fs",
    "staged_for_merge_fs",
    "staged_for_merge_pootle",
    "merged_from_fs",
    "merged_from_pootle",
]

FS_EXISTS = BOTH_EXIST + ["fetched_from_fs"]

STORE_EXISTS = BOTH_EXIST + ["added_from_pootle"]
```

File: tests/pootle_fs/display.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+
+import pytest
```
Comment (unho) [inline tests/pootle_fs/display.py]: Too many blank lines around this import.

**Comment by ta2-1** [comment]
lgtm

**Comment by unho** [comment]
lgtm, nits aside.

File: tests/pootle_fs/display.py [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+
+from pootle_fs.display import (
+    FS_EXISTS_ACTIONS, STORE_EXISTS_ACTIONS,
+    # FS_EXISTS_STATES, STORE_EXISTS_STATES,
```
Comment (unho) [inline tests/pootle_fs/display.py]: Is this commented for some reason?

File: pootle/apps/pootle_fs/display.py (hunk 1 above)
Comment (unho) [inline pootle/apps/pootle_fs/display.py]: The code changed, but I still think this suggestion applies to new code.

File: pootle/apps/pootle_fs/display.py [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+from pootle_fs.response import FS_RESPONSE
+from pootle_fs.state import FS_STATE
+
+
+FS_EXISTS_ACTIONS = [
+    "pulled_to_pootle", "pushed_to_fs", "fetched_from_fs",
+    "staged_for_merge_fs", "staged_for_merge_pootle",
+    "merged_from_fs", "merged_from_pootle"]
+STORE_EXISTS_ACTIONS = [
+    "pulled_to_pootle", "pushed_to_fs", "added_from_pootle",
```
Comment (unho) [inline pootle/apps/pootle_fs/display.py]: See https://github.com/translate/pootle/pull/4798#discussion_r66608428

**Comment by unho** [comment]
lgtm but please address nits.

File: tests/pootle_fs/display.py (hunk 3 above)
Comment (phlax) [inline tests/pootle_fs/display.py]: just because i havent written the  tests that need these yet

File: tests/pootle_fs/display.py (hunk 2 above)
Comment (unho) [inline tests/pootle_fs/display.py]: A single line before and a single line after is enough.

**Pull request merged**